doc/api_methods.md

```diff
@@ -70,14 +70,14 @@ Returns an array of layers in the map. The first is the base layer.
 
 ### vis.addOverlay(_options_)
 
-Adds an overlay to the map that can be either a zoom control, a tooltip or an infobox.
+Adds an overlay to the map that can be either a zoom control, a tooltip or an infowindow (hover action).
 
```

Comment:
I cannot view the example. 
What is the difference between infobox and infowindow exactly? Are they not related at all?